core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/TrafParallelClientScanner.java.tmpl

```diff
@@ -116,21 +116,22 @@ public class TrafParallelClientScanner extends AbstractClientScanner implements
     LOG.debug("Found "+locs.size()+" region(s).");
     if (locs.size() == 0) return;
 
+    int threads = parallelScaling>1.0 ?(int)Math.ceil(parallelScaling) : (int)Math.ceil(locs.size() * parallelScaling);
+
```

Comment:
Not sure I understand the logic behind this. Suppose locs.size() is 3. If parallelScaling is 1.1, then threads gets 2. If parallelScaling is 1.0, then threads gets 3. Seems odd that parallelScaling getting smaller should result in more threads.